### ISSUE TITLE
Problem with some syntax highlighting

### ISSUE THREAD

**Issue by BrachystochroneSD** [kopoli/robot-mode#10]
Some syntax highlight priority are problematic:

First found is that Keyword doesn't have font face, so if `font-lock-builtin-face` are used, they are shown as highlighted in some context, for example:

![image](https://github.com/kopoli/robot-mode/assets/34922853/d7725206-2022-4996-bd92-32b3caa6f502)

Some idea to fix this:
   - Give keyword a font-lock
   - `font-lock-builtin-face` should check for spaces or end-of-line after them so they dont 

Second is the comment font-lock that is overridden by the
```emacs-lisp
    ("^\\(Library\\|Resource\\|\\(Suite\\|Task\\|Test\\) \\(Setup\\|Teardown\\|Template\\|Timeout\\)\\|Variables\\):?\\s-*\\(.*\\)"
     (1 font-lock-preprocessor-face t) (4 font-lock-constant-face t))
```

Which cause this:
![image](https://github.com/kopoli/robot-mode/assets/34922853/729e21d0-93dd-4788-92ec-f5d0bb5eb961)

I guess, using `\\([^#]*\\)` intead of `\\(.*\\)`in the regexp should work for that

**Comment by kopoli** [comment]
Thanks for reporting these!
I added a pull-request which should address these issues.

**Comment by kopoli** [comment]
This is now fixed with the changes in #11 and merged in commit 127eabc08ace49a9a74ed44d14ed335aa7b1edd4
